### ISSUE TITLE
Beep sound when initializing AudioContext in ios device

### ISSUE THREAD

**Issue by sgwanlee** [software-mansion-labs/react-native-audio-api#228]
### Description

In simulator, there's no issue. However, when I initialize `AudioContext` in the iPhone, The beep sound is coming out.

Here's the code I used

```js
export default function HomeScreen() {
  const [isConnected, setIsConnected] = useState(false);
  const audioContextRef = useRef<AudioContext | null>(null);
  const clientRef = useRef<RealtimeClient>(
    new RealtimeClient({
      apiKey: process.env.EXPO_PUBLIC_OPENAI_API_KEY,
      dangerouslyAllowAPIKeyInBrowser: true,
    })
  );

  const connectConversation = async () => {
    console.log("connectConversation");
    if (!audioContextRef.current) {
      audioContextRef.current = new AudioContext();
    }
}
  
return (
    <SafeAreaView>
      <View style={{ padding: 10 }}>
        <TouchableOpacity
          onPress={isConnected ? disconnectConversation : connectConversation}
          style={{ padding: 4 }}
        >
          <Text style={{ color: "red" }}>
            {isConnected ? "Disconnect" : "Connect"}
          </Text>
        </TouchableOpacity>
      </View>
    </SafeAreaView>
  );

```

### Steps to reproduce

npm install
npx expo run:ios -d
Tap `connect` button

### Snack or a link to a repository

https://github.com/sgwanlee/react-natvie-audio-api-test

### React Native Audio API version

0.3.0-rc1

### React Native version

0.76.3

### Platforms

iOS

### JavaScript runtime

None

### Workflow

Expo Dev Client

### Architecture

None

### Build type

None

### Device

None

### Device model

iPhone 15 Pro, 18.1.1

### Acknowledgements

Yes

**Comment by michalsek** [comment]
We have just published 0.3.0-rc2, It should have the issue resolved :)

**Comment by sgwanlee** [comment]
It works! Thank you
